### ISSUE TITLE
"setBreakpoint" packet shall belong to the source actor

### ISSUE THREAD

**Issue by yurydelendik** [jimblandy/DebuggerDocs#26]
https://wiki.mozilla.org/Remote_Debugging_Protocol#Breakpoints shows that setBreakpoint shall be send to a thread actor, however it looks like it shall be sent to a source. Also, the location parameter does not contain the url field, but might contain the condition.

**Comment by jimblandy** [comment]
Yes, the protocol documentation should be updated. It should also be moved into the Firefox source tree, and broken up into separate pages by actor.

Since I'm the only person who has ever invested significant effort in this documentation, it doesn't seem like it's very valuable to the rest of the team, so I'm not expecting it to be updated.
